src/EPR.RegulatorService.Frontend.Web/Views/RegistrationSubmissions/RegistrationSubmissionDetails.cshtml

```diff
@@ -76,8 +76,16 @@
 
                 @await Component.InvokeAsync("SubmissionDetails", new { model = Model.SubmissionDetails })
 
-                @await Component.InvokeAsync("PaymentDetails", new { model = Model.PaymentDetails })
+                @* @await Component.InvokeAsync("PaymentDetails", new { viewModel = Model.PaymentDetails }) *@
```

Comment:
this will be removed as part of https://dev.azure.com/defragovuk/RWD-CPR-EPR4P-ADO/_workitems/edit/477227